website/AGENTS.md

```diff
@@ -1,3 +1,5 @@
 ## Checklist before committing code
 
 Run `CI=1 npm run test`, `npm run check-types`, `npm run format`.
+
```

Comment:
this was still unclear for me, I think explaining `Without CI=1, your test runner thinks it’s in development mode and keeps watching for file changes.
With CI=1, it knows it’s in automation mode, runs once, and exits cleanly` might be clearer